pom.xml

```diff
@@ -170,7 +170,7 @@ THE SOFTWARE.
       <dependency>
         <groupId>org.jenkins-ci.main</groupId>
         <artifactId>remoting</artifactId>
-        <version>3.15-20171116.114049-2</version>
+        <version>3.15</version>
```

Comment:
BTW https://github.com/jenkinsci/remoting/pull/245#pullrequestreview-85352695

Reply:
Fixed in https://github.com/jenkinsci/remoting/commit/7f7c94f2a532dd70761dddf4bc027f9fd81766a6